src/components/PageLayout/Navbar/Icons/NavbarIcon.vue

```diff
@@ -30,9 +31,16 @@
     components: {},
   })
   export default class NavbarIcon extends Vue {
+    public isShown = false;
+
     onShow() {
+      this.isShown = true;
       this.$emit('shown');
     }
+
+    onHide() {
+      this.isShown = false;
```

Comment:
Instead of using a ref to access this property, in Vue it is more idiomatic to emit an event with the updated evalue (eg, `this.$emit('shown', false)`) and then store that value on the component that uses it (NotificationIcon, in this case)

Reply:
fixed :D